python/BioSimSpace/Process/_gromacs.py

```diff
@@ -156,6 +159,9 @@ def __init__(self, system, protocol, exe=None, name="gromacs",
 
         # Initialise the PLUMED interface object.
         self._plumed = None
+		
+		# Set the path of Gromacs checkpoint file
+        self._prev_cpt = prev_cpt
```

Comment:
There is no type or value checking going on here. What should `prev_cpt` be (presumably a `str` type) and what happens when it's not?

Reply:
Thanks for the feedback @lohedges , I've added a type check and make sure it's a valid file.